stories/Select/SelectExample.js

```diff
@@ -1,118 +1,89 @@
+  { value: 'Value 3', label: 'Phasellus pretium orci sed odio tempus' },
+  { value: 'Value 4', label: 'Vestibulum bibendum ex vel augue porttitor sodales' },
+  { value: 'Value 5', label: 'Praesent quis elementum turpis' },
+];
 
-// const iconOptions = [
-//     { value: "video-1", label: "Video 1" },
-//     { value: "video-2", label: "Video 2" },
-//     { value: "video-3", label: "Video 3" }
-// ];
+export const groupOneOptions = [
+    { value: "option-1", label: "Opzione 1" },
+    { value: "option-2", label: "Opzione 2" }
+];
 
-// const multiOptions = [
-//     { value: "1", label: "Abruzzo" },
-//     { value: "2", label: "Basilicata" },
-//     { value: "3", label: "Calabria" },
-//     { value: "4", label: "Campania" },
-//     { value: "5", label: "Emilia Romagna" },
-//     { value: "6", label: "Friuli Venezia Giulia" },
-//     { value: "7", label: "Lazio" },
-//     { value: "8", label: "Liguria" },
-//     { value: "9", label: "Lombardia" },
-//     { value: "10", label: "Marche" },
-//     { value: "11", label: "Molise" },
-//     { value: "12", label: "Piemonte" },
-//     { value: "13", label: "Puglia" },
-//     { value: "14", label: "Sardegna" },
-//     { value: "15", label: "Sicilia" },
-//     { value: "16", label: "Toscana" },
-//     { value: "17", label: "Trentino Alto Adige" },
-//     { value: "18", label: "Umbria" },
-//     { value: "19", label: "Valle d'Aosta" },
-//     { value: "20", label: "Veneto" }
-// ];
+export const groupTwoOptions = [
+    { value: "option-3", label: "Opzione 3" },
+    { value: "option-4", label: "Opzione 4" }
+];
 
-// export const groupOneOptions = [
-//     { value: "option-1", label: "Opzione 1" },
-//     { value: "option-2", label: "Opzione 2" }
-// ];
+export const groupedOptions = [
+    {
+        label: "Gruppo 1",
+        options: groupOneOptions
+    },
+    {
+        label: "Gruppo 2",
+        options: groupTwoOptions
+    }
+];
 
-// export const groupTwoOptions = [
-//     { value: "option-3", label: "Opzione 3" },
-//     { value: "option-4", label: "Opzione 4" }
-// ];
+class SelectExample extends React.Component {
+  state = {
+    selectedOption: null,
+    placeholder: "Scegli una opzione",
+    isDisabled: false,
+    isSearchable: false,
+    isMulti: false,
+    isClearable: false,
+  };
+  handleChange = selectedOption => {
+    this.setState({ selectedOption });
+  };
+  render() {
+    const { selectedOption, disabled, search, multi, group, reset } = this.props;
+    let options = defaultOptions;
+        if (search) {
+            options = multiOptions;
+        } else if (multi) {
+          options = defaultOptions;
+        }else if (group) {
+          options = groupedOptions;
+        } 
 
-// export const groupedOptions = [
-//     {
-//         label: "Gruppo 1",
-//         options: groupOneOptions
-//     },
-//     {
-//         label: "Gruppo 2",
-//         options: groupTwoOptions
-//     }
-// ];
+    return (
+      <Select
```

Comment:
I would suggest to check [this](https://react-select.com/styles) out prior to consider this story complete, in order to style Select component in a way that is more similar to the [UI Kit directives](https://invis.io/WDSQQP543QV#/330541682_italia-DataEntry-Select).

Reply:
Yes, I have been through this earlier. Do you want to style it right now?